DEVELOPMENT.md

```diff
@@ -38,7 +38,7 @@ pip install -r test_requirements.txt
 
 and validate our tests are passing:
 ```
-SLS_OFFLINE=1 pytest
+SLS_OFFLINE=1 TESTING=1 DEPLOYMENT_STAGE=TEST pytest
```

Comment:
@benjamineac  maybe this file is redundant now? It's mostly duplicating out new README